### PR TITLE
5224 clean up tsk logical imager.exe packaging

### PR DESCRIPTION
**Pull request by jkho**

### CODE REVIEW

File: nbproject/project.properties [hunk 1]
```diff
@@ -6,8 +6,8 @@ app.name=${branding.token}
 ### if left unset, version will default to today's date
 app.version=4.12.0
 ### build.type must be one of: DEVELOPMENT, RELEASE
-build.type=RELEASE
-#build.type=DEVELOPMENT
+#build.type=RELEASE
+build.type=DEVELOPMENT
```
Comment (rcordovano) [inline nbproject/project.properties]: This change needs to be removed. We need this branch to be using the DEVELOPMENT build.type.

File: Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java [hunk 2]
```diff
@@ -41,18 +43,20 @@
...
         id = "org.sleuthkit.autopsy.configurelogicalimager.ConfigureLogicalImager"
 )
-@ActionRegistration(
-        displayName = "#CTL_ConfigureLogicalImager"
-)
+@ActionRegistration(displayName = "#CTL_ConfigureLogicalImager", lazy = false)
 @ActionReference(path = "Menu/Tools", position = 2000, separatorBefore = 1999)
 @Messages("CTL_ConfigureLogicalImager=Configure Logical Imager")
-public final class ConfigureLogicalImager implements ActionListener {
+public final class ConfigureLogicalImager extends CallableSystemAction {
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java]: This class needs to have action in the name. Let's call it  CreateLogicalImagerAction.

File: Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java [hunk 3]
```diff
@@ -41,18 +43,20 @@
...
+
+    private static final String DISPLAY_NAME = Bundle.CTL_ConfigureLogicalImager();
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java]: I think Will has changed the display name for this action so there should not be anything to do, but please coordinate with him.

File: Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java (hunk 2 above)
Comment (jkho) [reply Core/src/org/sleuthkit/autopsy/logicalimager/configuration/ConfigureLogicalImager.java]: @wschaeferB  PR will address this.

File: nbproject/project.properties (hunk 1 above)
Comment (jkho) [reply nbproject/project.properties]: done

**Comment by bcarrier** [comment]
This should also delete the .exe from the repo, right?

**Pull request merged**